tests/unit/debug_test.sh

```diff
@@ -120,7 +120,7 @@ function test_process_list()
   raw_input=''
   output=$(process_list "$raw_input" 1)
   ret="$?"
-  assert_equals_helper 'Return error:' "($LINENO)" "$ret" 22
+  assert_equals_helper 'Return error:' "($LINENO)" 22 "$ret"
```

Comment:
We can also take the opportunity to remove the parenthesis around `$LINENO` in the `assert_equals_occurences` **you've changed**. The parenthesis is unnecessary for `assert_equals_occurences` (differently from `assertEquals`, in which the parenthesis really make the output more readable).